frontend/src/components/Comments/UserComment/UserComment.test.js

```diff
@@ -0,0 +1,35 @@
+import { render, screen } from '@testing-library/react';
+import { UserComment } from './UserComment';
+import avatar1 from '../../../mocks/images/avatars/sample-avatar1.jpg';
```

Comment:
```suggestion
import avatar1 from 'mocks/images/avatars/sample-avatar1.jpg';
```
A działa coś takiego?